checkQC/qc_checkers/reads_per_sample.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+
+from checkQC.handlers.qc_handler import QCErrorFatal, QCErrorWarning
+
+
+def reads_per_sample(
+    self,
+    error_threshold,
+    warning_threshold,
+):
+    """
+    Check number of reads per sample
+    """
+    assert self.sequencing_metrics
+    assert error_threshold < warning_threshold
+
+    def _qualify_error(lane, number_of_samples, sample_id, sample_reads):
```

Comment:
```suggestion
    def _qualify_error(lane, number_of_samples, sample_id, cluster_count):
        sample_reads = cluster_count / 10**6
```
I find this a bit clearer: i.e one sees in a single place how sample_reads is defined. If you accept this, remember to update line 48 too.

Reply:
Yeah, that makes sense :+1: